actions/marketplace.ts

```diff
@@ -119,47 +120,48 @@ export function installPlugin(id: string) {
     };
 }
 
-// installApp installed an App using a given URL via the /apps install slash command.
+// installApp installed an App using a given URL a call to the `/install-listed` endpoint.
 //
-// On success, it also requests the current state of the plugins to reflect the newly installed plugin.
+// On success, it also requests the current state of the apps to reflect the newly installed app.
 export function installApp(id: string) {
     return async (dispatch: DispatchFunc, getState: GetStateFunc): Promise<boolean> => {
         dispatch({
             type: ActionTypes.INSTALLING_MARKETPLACE_ITEM,
             id,
         });
 
-        const state = getState() as GlobalState;
+        const callPath = '/install-listed';
+        const call: AppCall = {
+            path: callPath,
+        };
 
+        const expand: AppExpand = {
+            acting_user: '+summary',
+            locale: 'all',
+        };
+
+        const values: AppFormValues = {
+            app: {
+                label: id,
+                value: id,
+            },
+        };
+
+        const state = getState();
         const channelID = getCurrentChannelId(state);
         const teamID = getCurrentTeamId(state);
+        const context = createCallContext('apps', '/marketplace', channelID, teamID);
```

Comment:
Is `/marketplace` the location here? What is the location used in other form calls?

Reply:
@hanzei Yes I kind of made this new location up here. The location in other form calls would be something like `/command/jira/issue/create`. 0/5 on what the location should be here, or if we should include one at all

Reply:
Can we just not use a location at all?

Reply:
@hanzei I think there are tradeoffs between including or not including a location. I think we have the convention that all calls come from a location, but I think that can be avoided in this case. Do you see value in including the location? We can instead have it be an empty string.